### ISSUE TITLE
Die Spaltenbreite sollte automatisch so gewählt werden, dass der längste Dateiname in voller Länge angezeigt wird.

### ISSUE THREAD

**Issue by rhiinschnookele** [files-community/Files#16212]
### What feature or improvement do you think would benefit Files?

Die Spaltenbreite sollte automatisch so gewählt werden, dass der längste Dateiname in voller Länge angezeigt wird.

### Requirements

Die Spaltenbreite sollte automatisch so gewählt werden, dass der längste Dateiname in voller Länge angezeigt wird.

### Files Version

3.7.6.0

### Windows Version

10.0.22631.4169

**Comment by Josh65-2201** [comment]
Thanks for the feedback, is this the same as #9377

**Comment by yaira2** [comment]
@rhiinschnookele did you know that you can double click the resize bar to auto size the column?
<img width="217" alt="image" src="https://github.com/user-attachments/assets/62c7e059-3443-49ec-859a-9ccaa7f7f59d">

**Comment by rhiinschnookele** [comment]
Yes, I know that. But I would prefer it automatically. Just a wish. :-)

Am 17.09.2024 16:17 schrieb Yair: 

> @rhiinschnookele [1] did you know that you can double click the resize bar to auto size the column?
> image.png (view on web) [2] 
> 
> --
> Reply to this email directly, view it on GitHub [3], or unsubscribe [4].
> You are receiving this because you were mentioned.Message ID: ***@***.***>
 

Links:
------
[1] https://github.com/rhiinschnookele
[2]
https://github.com/user-attachments/assets/62c7e059-3443-49ec-859a-9ccaa7f7f59d
[3]
https://github.com/files-community/Files/issues/16212#issuecomment-2355978523
[4]
